### ISSUE TITLE
I get this error for rubygems

### ISSUE THREAD

**Issue by senthilnayagam** [dissonanz/aws-lambda-ruby#1]
START RequestId: 651364ed-716a-11e5-a0a9-9d75e3bbe613 Version: $LATEST
internal:gem_prelude:1:in `require': cannot load such file -- rubygems.rb (LoadError)
    from <internal:gem_prelude>:1:in`<compiled>'

END RequestId: 651364ed-716a-11e5-a0a9-9d75e3bbe613
REPORT RequestId: 651364ed-716a-11e5-a0a9-9d75e3bbe613  Duration: 112.07 ms Billed Duration: 200 ms     Memory Size: 128 MB Max Memory Used: 26 MB

**Comment by senthilnayagam** [comment]
I have not modified your ruby code

**Comment by dissonanz** [comment]
You're right, it doesn't work right now. The only binaries that are there are ruby and libruby.so.2.0, which is only enough to run `ruby -v` and doesn't allow to load any .rb files.

I'm working on including the entire ruby runtime (or a portion of it) so that it works as you would expect.

**Comment by senthilnayagam** [comment]
I have figured a way out for standalone ruby

https://github.com/senthilnayagam/LearnLambda

takes about 500ms for execution  and 15mb memory, binary is about 7mb so could upload directly
